pkg/adapter/containers_remote.go

```diff
@@ -852,3 +866,31 @@ func (r *LocalRuntime) Top(cli *cliconfig.TopValues) ([]string, error) {
 	}
 	return iopodman.Top().Call(r.Conn, ctr.ID(), descriptors)
 }
+
+// Port displays port information about existing containers
+func (r *LocalRuntime) Port(c *cliconfig.PortValues) ([]*Container, error) {
+	var (
+		containers []*Container
+		err        error
+	)
+	if !c.Latest && !c.All {
```

Comment:
Twice in the same PR??????

Reply:
well it is different ... remote vs local.  the logic is the same but calls differ.

Reply:
Well lets get a common function for remote and a common function for local and make all of the code useit.